app/2.0/start/toolbox/deploy.md

```diff
@@ -195,3 +195,10 @@ guide](https://www.firebase.com/docs/hosting/quickstart.html).
     The URL to your live site is listed in the output. You can also open
     the site in your default browser by running `firebase open hosting:site`.
 
+#### Optional: Edit your Firebase configuration to enable server push
```

Comment:
Can we verify whether the default build (or the default build when we launch) includes generating the push manifest? Might be that this should live in a different doc if it's more advanced (i.e., if the generated push manifest might need manual tweaking). Recommend we talk to the tools folks for a recommendation.